docs/internals/contributing/writing-code/accessibility.txt

```diff
@@ -0,0 +1,123 @@
+=============
+Accessibility
+=============
+
+The Django project is committed to being accessible to everyone
+in all possible ways.
+We work to conform to established accessibility standards,
+and also to meet our own targets for supporting specific input devices
+and assistive technologies.
+
+.. _accessibility-standards:
+
+Accessibility standards
+=======================
+
+We work to conform with the `Web Content Accessibility Guidelines`_ (WCAG),
+version 2.2, at the AA level.
+WCAG is the most established standard for web accessibility.
+AA-level guidelines are the most common legal compliance target worldwide.
+
+We also attempt to follow other best practices, such as:
+
+* WCAG 2.2 AAA-level guidelines.
+* The upcoming `WCAG 3.0`_ guidelines.
+* `Authoring Tool Accessibility Guidelines`_ (ATAG) 2.0 – guidelines
+  for software and services that are used to produce web content.
+
+To learn more about accessibility without diving straight into standards,
+we recommend `The A11Y Project`_, a community-driven effort
```

Comment:
An acronym definition of A11Y would be good here imo

Reply:
Hmmm interesting. Since this is the name of the website as they write it, I was hesitant to spell it out.